zkvm/src/predicate.rs

```diff
@@ -95,6 +95,18 @@ impl Predicate {
         }
     }
 
+    /// Creates a program with a random salt.
+    pub fn blinded_program<T: Into<Program>>(x: T) -> Self {
+        let salt: [u8; 16] = rand::random();
```

Comment:
Actually, `salt` is not a good name. It's private, so it should be called "pepper". But since we have exhausted our meme budget with a motocrab, maybe call it "blinding" (short for "blinding factor")?